go.mod

```diff
@@ -97,6 +97,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/patrickmn/go-cache v2.1.0+incompatible // indirect
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
This fix is already present in master branch, how come diff shows it as new addition

Reply:
I think merging master would remove it from the diff. But it shouldn't cause any problems as-is.

Reply:
I fixed this and put out the PR before that was merged into master I believe.